response_operations_ui/views/admin/manage_user_accounts.py

```diff
@@ -14,7 +14,7 @@
 from requests import HTTPError
 from structlog import wrap_logger
 
-from response_operations_ui.common import token_decoder
+from response_operations_ui.common import token_decoder, uaa
```

Comment:
Do we need to import all of uaa? do we not just need the function like the others. I think token_decoder might be the same

Reply:
Agreed.  Changed it.